src/lib.rs

```diff
@@ -934,19 +930,102 @@ impl DstLayout {
     }
 }
 
-/// A trait which carries information about a type's layout that is used by the
-/// internals of this crate.
+/// Implements [`KnownLayout`].
+///
+/// This derive analyzes various qualities of a type's layout that are needed
```

Comment:
```suggestion
/// This derive analyzes various properties of a type's layout that are needed
```

Alternatively, "aspects"?

Same below in the other doc comment.

Reply:
Done